rel/overlay/etc/default.ini

```diff
@@ -147,6 +147,27 @@ max_db_number_for_dbs_info_req = 100
 ; prevent non-admins from accessing /_all_dbs
 ; admin_only_all_dbs = true
 
+;;;; Moved from [httpd] ;;;;;
```

Comment:
Minor nit: since we're not using ';;;...' type headers anywhere else in the file, let's stick to just a plain comment, otherwise I think it looks inconsistent